files/usr/share/defaults/jeos-firstboot.conf

```diff
@@ -0,0 +1,6 @@
+# Example configuration file for jeos-firstboot
+
+# JEOS_LOCALE='en_US'
```

Comment:
There should be a line of documentation for every entry here.

Reply:
Indeed. Done